pages/workspace/publishing.mdx

```diff
@@ -0,0 +1,186 @@
+---
+layout: 'guide'
+title: 'Publishing'
+# abstract: 'Learn about how to publish your Tracking Plan to analytics platforms and APIs'
+---
+
+import Link from '../../components/Link';
+
+<p>
+  <img
+    src={require('../../images/publishing.svg')}
+    width="200px"
+    alt="Avo Publishing"
+/>
+
+</p>
+
+Publishing is where you can configure Avo to publish your Tracking Plan to your analytics platform, a production time validation tool or your own custom endpoint via webhook.
+
+Available publish integrations are currently:
+
+- <a href="#segment-protocols">Segment Protocols</a>
+- <a href="#mixpanel-lexicon">Mixpanel Lexicon</a>
+- <a href="#amplitude-govern">Amplitude Govern</a>
+- <a href="#webhook">Webhook</a>
+
+## Publishing configuration
+
+### Filtering events for publishing
+
+You can customize which events in Avo are published to which publish integrations by using the events filter. Events can be filtered by Sources, Destinations and Tags. For example the following publish integration will only publish events sent from Web to Amplitude.
+
+<center>
+  <img
+    src={require('../../images/publish_filter.png')}
+    width="100%"
+    alt="Publish integration filter to publish only events sent from Web to Amplitude"
+  />
+</center>
+
+### Auto publishing
+
+When Auto Publishing is enabled all changes made on a branch will be published to the integration when the branch is merged.
+
+<center>
+  <img
+    src={require('../../images/auto_publishing.png')}
+    width="100%"
+    alt="Publish integration filter to publish only events sent from Web to Amplitude"
+  />
+</center>
+
+# Publishing integrations
+
+The following publish integrations are available in Avo
+
+- <a href="#segment-protocols">Segment Protocols</a>
+- <a href="#mixpanel-lexicon">Mixpanel Lexicon</a>
+- <a href="#amplitude-govern">Amplitude Govern</a>
+- <a href="#webhook">Webhook</a>
+
+They enable your Tracking Plan to be
+
+- published with a push of a button
+- published whenever a branch is merged with Auto Publishing
+- downloaded from your Avo workspace on relevant format
+
+<img
+  src={require('../../images/publishing_protocols.svg')}
+  alt="Segment"
+  style={{ margin: '32px 0 -32px 0' }}
+/>
+
+## <a name="segment-protocols"></a> Segment Protocols
+
+The publish integration to [Segment Protocols](https://segment.com/docs/protocols) allows for your Tracking Plan to be published into Segment Protocols whenever branches are merged in your workspace or with a push of a button.
+
+### Configure the Protocols integration
+
+To enable the direct publish integration with Segment Protocols, you need to configure it with the following parameters from Segment. This step is not neccassary if you only wish to download the Protocols JSON file from the Avo workspace.
```

Comment:
I'd love to know more about this step being unnecessary. What would I dow with the downloaded Protocols JSON file? Should we be explicit about there being an automatic way to use this integration and a manual way? Perhaps have an automatic section first and then end with a manual section?

Reply:
Or maybe in this case just skip the mentioning of the manual step?